### DISCUSSION TITLE
DLL/Library files for C# or VB.NET

### DISCUSSION THREAD

**Discussion by RaB-dK** [Yaskawa-Global/YMConnect#18] [General]
Hello,

Is it possible to to compile DLL/LIB files for C# or VB.NET ?

I made the simple "Hello World" C++ console app and it works fine with a YRC1000 controller. But as I am making a Windows HMI/GUI app I do not see the purpose of using C++ console.

I have tried making C++ app with winforms, but i could not get it to work. 

FYI:; I have experience with C# or VB.NET but NOT C++

**Comment by aseamans** [comment]
Hello,
We are currently working on a C# version. We don't have any plans for VB.NET at this time. 
Thank you.

**Reply by RaB-dK** [reply]
Hello,

Thank you for your reply. Do you know when the C# version will be released?

**Reply by aseamans** [reply]
Its currently in development but we currently don't have a solid time frame.
